### PR TITLE
Add lmdb format dataset conversion tools

### PR DESCRIPTION
**Pull request by OneYearIsEnough**
This excellent repo supports lmdb format dataset in recognition tasks, but it doesn't have any tools which can convert the icdar2015 format dataset to lmdb dataset format, so i wrote it in **ppocr/utils** dir.
I've tested all the functions in my script, they work well in recognition tasks.(e.g. **rec_data_lesson_demo** dataset)

### CODE REVIEW

**Comment by OneYearIsEnough** [comment]
有大佬帮我review 下嘛😅？有问题的话我再改改~

File: ppocr/utils/make_lmdb.py [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+    return True
+
+
+def writeCache(env, cache):
+    with env.begin(write=True) as txn:
+        for k, v in cache.items():
+            txn.put(k.encode('utf-8'), v)
+
+
+def convert2lmdb(data_root_dir, label_file_path, lmdb_out_dir, is_check=False):
```
Comment (WenmuZhou) [inline ppocr/utils/make_lmdb.py:39]: 1. 弄成argparse的形式，通过命令行传参，label_file_path的参数需要支持列表
2. 放到ppocr/contrib目录下，添加readme说明文件的用法和功效

**Comment by Evezerest** [comment]
感谢贡献，建议新建项目后将项目链接PR到社区文档

File: ppocr/utils/make_lmdb.py (hunk 1 above)
Comment (OneYearIsEnough) [reply ppocr/utils/make_lmdb.py:39]: 已修改  
@Evezerest 说要单独开一个repo……
我放在了 https://github.com/OneYearIsEnough/PaddleOCR-Recog-LmdbDataset-Conversion 这个项目里，麻烦审核下哈

Comment (WenmuZhou) [reply ppocr/utils/make_lmdb.py:39]: 没问题

**Pull request closed without merging**